docs/user/ppl/cmd/describe.rst

```diff
@@ -16,9 +16,12 @@ Description
 
 Syntax
 ============
-describe <index>
+describe <catalog>.<index>
```

Comment:
Should we enforce schema (`default`?) in full table name? Because:

1. we already add standard `information_schema` in PPL;
2. it would be breaking change in future